docs/Symbols.md

```diff
@@ -22,5 +26,144 @@ The following are also listed:
 
 ![An XML document that references two grammars. The referenced grammars are listed in the outline](./images/Symbols/ShowReferencedGrammars.png)
 
-This option has no effect if symbols are disabled through `xml.symbols.enabled`.
-The displayed symbols are affected by `xml.symbols.maxItemsComputed`
+This option has no effect if symbols are disabled through [xml.symbols.enabled](#xmlsymbolsenabled).
+The displayed symbols are affected by [xml.symbols.maxItemsComputed](#xmlsymbolsmaxitemscomputed)
+
+## xml.symbols.filters
+
+By default Outline (symbols) display `DOM elements`, `processing instruction` and `DTD element, entity, attribute list declarations`.
+
+The DOM attributes and text nodes are not displayed in the outline. Indeed to ensure good performance, symbols must be limited; displaying attributes and text nodes in the outline could generate a lot of symbols.
+
+Here a sample Outline with maven `pom.xml`:
+
+![Outline of pom.xml without filter](images/Symbols/SymbolsPOMWithoutFilter.png)
```

Comment:
The images have white borders, maybe try to crop them?

Reply:
fixed

Reply:
They still have them: 
![Screenshot from 2020-10-27 13-50-46](https://user-images.githubusercontent.com/22376627/97341460-78a37180-185b-11eb-971a-d84336b22003.png)

Reply:
Is it better now? I have update the show referenced grammar too to add red square. If you don't like it, please tell me.

Reply:
Looks good!